platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SOCSpaceSearchTestIT.java

```diff
@@ -77,4 +86,75 @@ public void test01_SearchSpace() {
 
   }
 
+  @BugInPLF("PLF-8006")
```

Comment:
To be removed since this issue has been resolved.

Reply:
Done.